From e1304ddf3e45ec6529a9819bc807bb9b6e7c0747 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Sat, 28 Sep 2019 18:03:41 +0200
Subject: [PATCH 077/351] input: cyttsp4: Faster recovery from failed wakeup
 (HACK)

For some reason on Pocket Book Touch Lux 3, sending and I2C
data, doesn't wake up the device, and it always times out.

Lower the timeout, so that reset is done ASAP after wakeup.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 drivers/input/touchscreen/cyttsp4_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/touchscreen/cyttsp4_core.c b/drivers/input/touchscreen/cyttsp4_core.c
index b9e644cac606..1c890b4df28d 100644
--- a/drivers/input/touchscreen/cyttsp4_core.c
+++ b/drivers/input/touchscreen/cyttsp4_core.c
@@ -26,7 +26,7 @@
 #define CY_CORE_SLEEP_REQUEST_EXCLUSIVE_TIMEOUT	5000
 #define CY_CORE_MODE_CHANGE_TIMEOUT		1000
 #define CY_CORE_RESET_AND_WAIT_TIMEOUT		500
-#define CY_CORE_WAKEUP_TIMEOUT			500
+#define CY_CORE_WAKEUP_TIMEOUT			50
 
 #define CY_CORE_STARTUP_RETRY_COUNT		3
 
-- 
2.34.0

